app/src/main/java/org/simple/clinic/login/pin/LoginPinScreen.kt

```diff
@@ -51,7 +53,7 @@ class LoginPinScreen(context: Context, attrs: AttributeSet) : RelativeLayout(con
 
     TheActivity.component.inject(this)
 
-    Observable.mergeArray(screenCreates(), pinTextChanges(), submitClicks(), backClicks(), otpReceived())
+    Observable.mergeArray(screenCreates(), pinTextChanges(), submitClicks(), backClicksFromUi(), backClicksFromSystemBack(), otpReceived())
```

Comment:
`fromUi` sounds weird 😄

Reply:
I propose merging `backClicksFromUi()` and `backClicksFromSystemBack()` into just `backClicks()`. Wdyt?